recipes/_bash.rb

```diff
@@ -106,3 +106,17 @@
     export PATH="/usr/local/bin:$PATH"
   EOH
 end
+
+if solaris_11?
```

Comment:
The extra RBAC stuff on Solaris 11 requires that both the current shell of the build user (`/usr/bin/bash`) and the new one (`/usr/local/bin/bash`) be listed in the `/etc/shells` file.. which it doesn't have by default.. It also requires that both the old and new shell files actually exist (as far as I can tell). So setting the build user's shell can only be done after /etc/shells is populated and after bash has been compiled/installed.